climate.py

```diff
@@ -186,6 +199,38 @@ def fan_modes(self):
         import wideq
         return [v for k, v in FAN_MODES.items() if wideq.ACFanSpeed[k].value in self._ac.model.value('SupportWindStrength').options.values()]
 
+    @property
+    def swing_mode(self):
+        # try to find out if the (initial) state matches a known state actually
+        if self._swing_mode == "UNKNOWN":
+            for k, v in SWING_MODES.items():
+                if v[1] == self._state.horz_swing and v[2] == self._state.vert_swing:
+                    self._swing_mode = k
+        
+        if self._swing_mode == "UNKNOWN":
+            return "Unknown"
```

Comment:
```suggestion
                    self._swing_mode = k
                    break
            else:
                return "Unknown"
```

Here's a somewhat simpler way to do it.

Reply:
wasn't aware of for/else, thanks